activejob/CHANGELOG.md

```diff
@@ -22,5 +22,13 @@
 
     *Evgenii Pecherkin*, *Rafael Mendonça França*
 
+*   Add randomness to :exponentially_longer
+
+    Using ActiveJob::Exceptions.retry_on with :exponentially_longer now
+    uses a random amount of jitter in order to prevent the [thundering herd effect.](https://en.wikipedia.org/wiki/Thundering_herd_problem).  Default jitter amount of 0.15 but overridable via the `:jitter` option
```

Comment:
This suggests that jitter is a duration, or some number with units.

Can we explain how jitter is applied to the wait time?